docs/core-concepts/core-sdk/cli/overview.md

```diff
@@ -0,0 +1,121 @@
+
+### How does it work?
+As we saw in the short sample [above](#a-short-sample), there is a pattern in the way you use the CLI tools. Within that pattern, we can 
+identify three main pieces of each command:
+
+1. The driver ("dotnet")
+2. The command, or "verb"
+3. Command arguments
+
+Let's dig into more details on each of the above.
+
+### Driver
+The driver is named `dotnet`. It is the first part of what you invoke . The driver has two responsibilities:
+
+1. Executing IL code
+2. Executing the verb
+
+Which of the two things it does is dependent on what is specified on the command line. In the first case, you would 
+specify an IL assembly that `dotnet` would run similar to this: `dotnet /path/to/your.dll`. 
+
+In the second case, the driver attempts to invoke the specified command. This will start the CLI command execution 
+process. First, the driver will determine the version of the tooling that you want. You can specify the version in the 
+`global.json` file using the `sdkVersion` property. If that is not available, the driver will find the latest version
+of the tools that is installed on disk and will use that version. Once the version is determined, it will execute the 
+command. 
+
+### The "verb"
+The verb here is simply a command that performs an action. `dotnet build` will build your code. `dotnet publish` will publish 
+your code. The verb is implemented as a console application that is named per convention: `dotnet-{verb}`. All of the 
+logic is implemented in the console application that represents the verb. 
+
+### The arguments
+The arguments that you pass on the command line are the argument to the actual verb/command being invoked. 
+For example, when you type `dotnet publish --output publishedapp` the `--output` argument is passed to the 
+`publish` command. 
+
+## Type of application portability
+CLI enables applications to be portable in two main ways:
+
+1. Completely portable application that can run anywhere .NET Core is installed
+2. Self-contained applications
+
+You can learn more about both of these in the [application types overview](../../app-types.md) topic. 
+
+## Migration from DNX
+If you used DNX in RC1 of .NET Core, you may be wondering what happened to it and how do these new tools
+relate to the DNX tools. In short, the DNX tools have been replaced with the .NET Core CLI tools in RC2. 
+If you have existing projects or are just wondering how the commands map, you
+can use the [DNX to CLI migration document](../../dnx-migration.md) to get all of the details. 
+
+## Extensibility
+Of course, not every tool that you could use in your workflow will be a part of the core CLI tools. However, .NET Core 
+CLI has an extensibility model that allows you to specify additional tools for your projects. You can find out more 
+in the [extensibility document](extensibility.md). 
+
+## More resources
+This was a short overview of the most salient features of the CLI. You can find out more by using the reference and 
```

Comment:
Cultural bias here I guess. Salient in Brazil (since it's a Latin word) is mostly used in a pejorative manner when referring to women. And I believe @cartermp also had issues with this word before. :laughing: